packages/core/src/data-module/TimeSeriesDataModule.spec.ts

```diff
@@ -15,6 +15,7 @@ import Mock = jest.Mock;
 const { EMPTY_CACHE } = caching;
 
 const { propertyId: PROPERTY_ID, assetId: ASSET_ID } = toSiteWiseAssetProperty(DATA_STREAM.id);
+type SiteWiseDataStreamQuery = { assets: { assetId: string, properties: { refId?: string; propertyId: string }[] } [] } & DataStreamQuery;
```

Comment:
is there no use for AssetQuery / PropertyQuery anywhere else internally? I know we don't want them to be a part of the public api, but would those still be useful?

Reply:
they are exposed within the source-iotsitewise package

Reply:
ahh, and we couldn't use them here because it would lead to a circular reference then

Reply:
Also these tests aren't really testing the sitewise data source even though they look similar. If we were to invest more time into it I would make a simplified query format more detached from SiteWise